zipkin/src/test/java/com/linecorp/armeria/server/tracing/HttpTracingServiceTest.java

```diff
@@ -57,6 +59,22 @@
 
     private static final String TEST_METHOD = "hello";
 
+    @After
+    public void tearDown() {
+        Tracing.current().close();
```

Comment:
What does this do?

Reply:
clears the state caused when we initialize tracers.. this was in the HttpTracingClient file.. I copied it. It is slightly less crap than holding a ref to each tracing instance just to close them.